workshops/1.md

```diff
@@ -57,64 +58,68 @@ We agreed on disabling GitHub issues for internal projects and moving all issue
 
 A few other common anti-patterns:
-- Form fields without labels
-- Text only buttons
-- Square buttons (rounded corners please)
+- Form fields without labels.
+- Text only or textless buttons (with exceptions).
+- Square buttons (rounded corners please).
+- `X` for closing modals.
+- Assets with differing corner radii.
+- Over-complicated custom navigation flows
+- Features hidden behind gestures or inscrutable icons (the Snapchat effect).
 
-We discussed that doing more wireframing would be very helpful to check for early design inconsistencies or flaws. Jumping straight into high fidelity designs might not be the best idea.
+We discussed that doing more wireframing would be very helpful to check for early inconsistencies or flaws before there's too much work put into the design. Jumping straight into high fidelity designs might not be the best idea. More so for larger projects.
 
-We need to respect the platforms, iOS, Android and web developers need to be aligned to make sure that designs don't just get recycled into new platforms but instead that they are optimized for each platform.
+We need to respect the platforms, iOS, Android and web developers need to be aligned to make sure that designs don't just get recycled into new platforms, but instead that they are optimized for each platform. In other words, iOS should look and feel like iOS (even if a lot of it is custom or branded), Android should look and feel like Android, and Web is everything goes?
 
-We discussed that would be great to make a Sketch plugin that would export the colors and fonts used in the apps. Basically a button on Sketch that would create a Theme.swift file. One of the reasons that we want this is because in designs is common to have colors with some transparency instead of flat colors, or backgrounds where the parent group is the one with the transparency, this is a mess to work with and having this tool would help a lot.
+We discussed that it would be great to make a Sketch plugin that would export the colors and fonts used in the app. Basically a button in Sketch that would create a Theme.swift file. One of the reasons that we want this is because in designs it's common to have colors with some transparency instead of flat colors, or backgrounds where the parent group is the one with the transparency, this is a mess to work with and having this tool would help a lot. We want to avoid real-time composition as much as possible, as it's harder to ensure the result is the wanted one, it's also more CPU/GPU expensive and can slow down animations.
```

Comment:
>We want to avoid real-time composition as much as possible, as it's harder to ensure the result is the wanted one, it's also more CPU/GPU expensive and can slow down animations.

Are you referring to transparency there? I think the performance hit for this is too small to be important at all.